system-tests/package.json

```diff
@@ -30,7 +30,7 @@
     "@packages/ts": "0.0.0-development",
     "@storybook/testing-vue3": "0.0.1",
     "@types/chai": "4.2.15",
-    "@types/mocha": "9.1.0",
+    "@types/mocha": "8.0.3",
```

Comment:
The version of `mocha` in this `package.json` is `7.x`.

I'm not too sure on the best way to do this - should we standardize a single version of Mocha in the entire code base? Maybe 8.x?